packages/cli/test/functional/test_site_special_tags/_markbind/variables.md

```diff
@@ -0,0 +1,4 @@
+<variable name="example">
+To inject this HTML segment in your MarkBind files, use {{ example }} where you want to place it.
```

Comment:
Note: for some reason there's no `variables.md` in this and the other site above, but we should include it to avoid unnecessary warning.